doc/rules/php_unit/php_unit_data_provider_static.rst

```diff
@@ -32,10 +46,29 @@ Example #1
    +    public static function provideSomethingCases() {}
     }
 
+Example #2
+~~~~~~~~~~
+
+With configuration: ``['force' => true]``.
+
+.. code-block:: diff
+
+   --- Original
+   +++ New
+    <?php
+    class FooTest extends TestCase {
+        /**
+         * @dataProvider provideSomethingCases
+         */
+        public function testSomething($expected, $actual) {}
+   -    public function provideSomethingCases() { $this->getData(); }
+   +    public static function provideSomethingCases() { $this->getData(); }
+    }
+
 Rule sets
 ---------
 
 The rule is part of the following rule set:
 
 @PHPUnit100Migration:risky
-  Using the `@PHPUnit100Migration:risky <./../../ruleSets/PHPUnit100MigrationRisky.rst>`_ rule set will enable the ``php_unit_data_provider_static`` rule.
+  Using the `@PHPUnit100Migration:risky <./../../ruleSets/PHPUnit100MigrationRisky.rst>`_ rule set will enable the ``php_unit_data_provider_static`` rule with the default config.
```

Comment:
I wonder if we can have the new option used in some ruleset. I have a feeling that if we do not, the ppl will not be aware about this option and not using it.

[no good answer to this, myself]

Reply:
Let's for a start do it in `@PhpCsFixer:risky`, but in separate PR as there are quite few changes (including sorting of function that makes diff big).

Reply:
Adding `force` option to `@PHPUnit100Migration:risky` might have more sense as without it the migration might not be successful (with it also as it will break the test suite, but will make it closer to successful migration).